### DISCUSSION TITLE
After changes to 'users' collection, unable to use OAuth2

### DISCUSSION THREAD

**Discussion by square-foot** [pocketbase/pocketbase#6043] [Q&A] [answered]
Hi

NOTE: I am using Pocketbase version 0.22.22 

My application login using OAuth2 was working quite fine. But now it is no longer allowing OAuth2.  I am trying to figure out what I may have done wrong. I checked that the OAuth2 toggle is switched on in the user's collection's options (the one that comes on the right side in the browser in the admin UI)  I also checked if I had accidentally switched off all the providers. But I haven't. Four of them are enabled: Google, Discord, Twitter and Github

Then I found that this gives an empty object: 
const result = await pb.collection('users').listAuthMethods();  

So the only plausible explanation I can think of is that this possibly happened after I added some more fields to the users collection.  I am not sure, though. What could be the reason? Would love to know how to rectify it. 

Thanks for a great product.

**Comment by ganigeorgiev** [comment]
~Please make sure that you are using JS SDK v0.22+ (see the [JS SDK CHANGELOG](https://github.com/pocketbase/js-sdk/blob/master/CHANGELOG.md)).~

If you are using PocketBase v0.22, then the last JS SDK compatible version should be v0.21.5 (see the above changelog).

**Comment by square-foot** [comment]
Thanks for the quick reply. I tried using this 

https://cdn.jsdelivr.net/gh/pocketbase/js-sdk@0.22.1/dist/pocketbase.umd.js

and also this 

https://cdn.jsdelivr.net/gh/pocketbase/js-sdk@0.22/dist/pocketbase.umd.js

But the error still persists

**Reply by ganigeorgiev** [reply, marked as answer]
Again, please read the JS SDK changelogs. PocketBase v0.22 is compatible only with max JS SDK v0.21.5.

JS SDK v0.22+ works only with PocketBase v0.23+.

**Reply by square-foot** [reply]
Thank you very much! That worked.
